x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/analytics_layout.test.tsx

```diff
@@ -47,18 +51,20 @@ describe('AnalyticsLayout', () => {
       </AnalyticsLayout>
     );
 
-    expect(wrapper.find(FlashMessages)).toHaveLength(1);
     expect(wrapper.find(LogRetentionCallout)).toHaveLength(1);
+    expect(getPageHeaderActions(wrapper).find(LogRetentionTooltip)).toHaveLength(1);
+    expect(getPageHeaderChildren(wrapper).find(AnalyticsFilters)).toHaveLength(1);
 
-    expect(wrapper.find(AnalyticsHeader).prop('title')).toEqual('Hello');
+    expect(getPageTitle(wrapper)).toEqual('Hello');
     expect(wrapper.find('[data-test-subj="world"]').text()).toEqual('World!');
+
+    expect(wrapper.prop('pageChrome')).toEqual(['Engines', 'some-engine', 'Analytics']);
```

Comment:
[non-blocking] It was a little confusing seeing that a `pageChrome` parameter accepted breadcrumbs as a parameter. I would have expected the parameter to be named something like `breadcrumbs`.

Reply:
Don't forget it's setting the document title too, which is very important 😄 That's why it's not just `breadcrumbs`, although maybe I should clarify the typing on that as well